openapi.yaml

```diff
@@ -526,6 +526,23 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/GenericApiResponse"
+  /deleteTopicRequests:
+    post:
+      operationId: deleteTopicRequests
+      requestBody:
```

Comment:
It seems that the definition of this endpoint implies that the `topicid` is actually passed as a query parameter, not as a `body`: https://github.com/aiven/klaw/blob/main/core/src/main/resources/openapi.yaml#L1086

So it should probably look more like this endpoint: https://github.com/aiven/klaw/blob/7552c6e924fc9094f461598a51ab8bdfd1783c4c/openapi.yaml#L340-L359

This has of course consequences on how we implement things in `topic-api.ts`. We can omit the payload argument to `post` and pass the absolute pathname + query param https://github.com/aiven/klaw/blob/7552c6e924fc9094f461598a51ab8bdfd1783c4c/coral/src/services/api.ts#L324-L325

And then it looks more like this `get` endpoint: https://github.com/aiven/klaw/blob/7552c6e924fc9094f461598a51ab8bdfd1783c4c/coral/src/domain/requests/requests-api.ts#L5-L11

Reply:
(incidentally, `execAclRequest` is the "old way" of doing `request/approve` for ACL requests ^^)